src/polyfem/State.cpp

```diff
@@ -1413,20 +1415,17 @@ namespace polyfem
 							  resolve_output_path(args["output"]["paraview"]["file_name"]));
 			}
 
-			Eigen::VectorXd c_sol;
-			init_transient(c_sol);
-			RhsAssembler &rhs_assembler = *(step_data.rhs_assembler);
-
 			if (formulation() == "NavierStokes")
-				solve_transient_navier_stokes(time_steps, t0, dt, rhs_assembler, c_sol);
+				solve_transient_navier_stokes(time_steps, t0, dt);
 			else if (formulation() == "OperatorSplitting")
-				solve_transient_navier_stokes_split(time_steps, dt, rhs_assembler);
-			else if (problem->is_scalar() || assembler.is_mixed(formulation()))
```

Comment:
Seems like this includes Laplacian, Helmholtz, and Bilaplacian but also IncompressibleLinearElasticity and Stokes which are tensor problems.